source/administration/changelog.md

```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.2
+
+### Highlights
+
+#### Embed Mattermost in Other Apps (Beta)
+ - Added support for extensions, which allow you to embed Mattermost in other apps and websites via OAuth 2.0.
+ - A sample extension for Chrome [is here](https://github.com/mattermost/mattermost-chrome-extension). For documentation on how to write an extension, see here. // XXX Add a link to doc when ready
+
+#### Plugins
+ - Breaking changes to the plugins framework introduced. To migrate your existing plugins to be compatible with Mattermost 5.2 and later, see our [migration guide](https://developers.mattermost.com/extend/plugins/migration/).
+ - Added support to add/delete and enable/disable plugins via the CLI.
+ - See our [demo plugin](https://github.com/mattermost/mattermost-plugin-demo) that demonstrates the capabilities of a Mattermost plugin. For a starting point to write a Mattermost plugin, see our [sample plugin](https://github.com/mattermost/mattermost-plugin-sample).
+ 
+#### Searching Archived Channels
+ - Added ability to search for archived channels.
+ 
+#### Romanian Language
+ - Added support for Romanian language.
+
+### Improvements
+
+#### Web User Interface (UI)
+ - Added experimental custom default channels.
```

Comment:
Is this a config? If so it isn't under the config changes section. Do you have a link to more info on this as I'm not familiar with this change

Reply:
@esethna Here the PR: https://github.com/mattermost/mattermost-server/pull/9068.

Reply:
Ah, this needs to be added to config changes compatibility section

Reply:
https://github.com/mattermost/mattermost-server/pull/9068/files#diff-2279f86634da3f787f279961af76713aR1108

Reply:
@esethna It's not in the config file though https://github.com/mattermost/mattermost-server/blob/master/config/default.json.